.travis.yml

```diff
@@ -23,7 +23,7 @@ script:
 - if [[ $TRAVIS_REPO_SLUG == "kubevirt/kubevirt" ]]; then make goveralls; else make test; fi
 - make apidocs
 - make client-python
-- make manifests DOCKER_TAG=$TRAVIS_TAG # falls back to latest if not on a tag
+- make manifests DOCKER_PREFIX="docker.io/kubevirt" DOCKER_TAG=$TRAVIS_TAG # falls back to latest if not on a tag
```

Comment:
I think you will also have to modify hack/config-default.sh (to recognize DOCKER_PREFIX) and the Makefile (to pass DOCKER_PREFIX to the build container).

Reply:
Meh - Thanks. I knew this had to be done somewhere. I just lost track. 
Fixed.

Reply:
Now only the makefile target needs to be adjusted so that the build inside the docker container can see the environment variable.